docker-php-ext-configure

```diff
@@ -2,13 +2,12 @@
 set -e
 
 ext="$1"
-extDir="/usr/src/php/ext/$ext"
-if [ -z "$ext" ] || ! [ -d "$extDir" ]; then
+if [ -z "$ext" ] || ! grep -qE "^$ext$" /usr/src/available-php-exts; then
```

Comment:
You're going to hate me a little, but I think `/usr/src/php-available-exts` might be a better name for this file (for symmetry between `/usr/src/php`, `/usr/src/php.tar...`, etc).